benchmark_runner/main/environment_variables.py

```diff
@@ -54,10 +56,11 @@ def __init__(self):
         self._environment_variables_dict['pin_node2'] = EnvironmentVariables.get_env('PIN_NODE2', '')
 
         # ElasticSearch
-        self._environment_variables_dict['elasticsearch'] = EnvironmentVariables.get_env('ELASTICSEARCH', '')
-        self._environment_variables_dict['elasticsearch_port'] = EnvironmentVariables.get_env('ELASTICSEARCH_PORT', '')
-        self._environment_variables_dict['elasticsearch_user'] = EnvironmentVariables.get_env('ELASTICSEARCH_USER', '')
-        self._environment_variables_dict['elasticsearch_password'] = EnvironmentVariables.get_env('ELASTICSEARCH_PASSWORD', '')
+        # ElasticSearch
+        self._environment_variables_dict['elasticsearch'] = os.environ.get('ELASTICSEARCH', '')
+        self._environment_variables_dict['elasticsearch_port'] = os.environ.get('ELASTICSEARCH_PORT', '')
+        self._environment_variables_dict['elasticsearch_user'] = os.environ.get('ELASTICSEARCH_USER', '')
+        self._environment_variables_dict['elasticsearch_password'] = os.environ.get('ELASTICSEARCH_PASSWORD', '')
```

Comment:
Why this change?  That prevents passing these arguments on the command line.

Reply:
Fixed